### PR TITLE
[ACS-8201] Knowledge Retrieval - getting AI response for one or more selected files

### PR DESCRIPTION
**Pull request by jacekpluta**
**Please check if the PR fulfills these requirements**

> - [x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [x] Tests for the changes have been added (for bug fixes / features)
> - [x] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [x] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)
https://hyland.atlassian.net/browse/ACS-8201

**What is the new behaviour?**

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [ ] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: docs/content-services/services/agent.service.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+Title: Agent service
+Added: v6.9.1
```
Comment (MichalKinas) [inline docs/content-services/services/agent.service.md:3]: I think next version of ADF we're planning to release would be `7.0.0-alpha.3`

File: docs/content-services/services/search-ai.service.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
+---
+Title: Search Ai service
+Added: v6.9.1
```
Comment (MichalKinas) [inline docs/content-services/services/search-ai.service.md:3]: Same here

File: docs/versionIndex.md [hunk 3]
```diff
@@ -12,6 +12,7 @@ backend services have been tested with each released version of ADF.
 
 ## Versions
 
+-   [v6.9.1](#v691)
```
Comment (MichalKinas) [inline docs/versionIndex.md:15]: Should be adapted

File: docs/content-services/services/agent.service.md
Comment (MichalKinas) [inline docs/content-services/services/agent.service.md:1]: I also wonder for both of those files, should we mention in the docs that in order to actually use them additional ACS module should be installed? wdyt?

File: lib/core/src/lib/dialogs/unsaved-changes-dialog/unsaved-changes-dialog.component.ts [hunk 4]
```diff
@@ -15,22 +15,55 @@
...
+        this.dialogData = {
+            headerText: this.data?.headerText ?? 'CORE.DIALOG.UNSAVED_CHANGES.TITLE',
+            descriptionText: this.data?.descriptionText ?? 'CORE.DIALOG.UNSAVED_CHANGES.DESCRIPTION',
+            confirmButtonText: this.data?.confirmButtonText ?? 'CORE.DIALOG.UNSAVED_CHANGES.DISCARD_CHANGES_BUTTON',
+            checkboxText: this.data?.checkboxText ?? ''
+        };
+    }
+
+    /**
+     * Sets 'unsaved_ai_changes__modal_visible' checked state (true or false string) as new item in local storage.
```
Comment (MichalKinas) [inline lib/core/src/lib/dialogs/unsaved-changes-dialog/unsaved-changes-dialog.component.ts:62]:
```suggestion
     * Sets 'unsaved_changes__modal_visible' checked state (true or false string) as new item in local storage.
```

File: lib/js-api/src/api/content-rest-api/api/agents.api.ts [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+import { AgentPaging } from '../model/agentPaging';
+import { BaseApi } from '../../hxi-connector-api/api/base.api';
+
+/**
+ * Agents Api.
+ */
+export class AgentsApi extends BaseApi {
+    /**
+     * Gets all agents.
```
Comment (MichalKinas) [inline lib/js-api/src/api/content-rest-api/api/agents.api.ts:26]: Same here do we want to add a note that it requires additional ACS module?

**Review by MichalKinas — APPROVED** [review]
Couple of smaller comments but overall it LGTM

**Review by AleksanderSklorz — APPROVED** [review]

**Pull request merged**